samwise.js

```diff
@@ -1,39 +1,39 @@
-const capitalizedString = string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.